com.github.qarmin.szyszka.yaml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.qarmin.szyszka
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: szyszka
+finish-args:
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--filesystem=host"
```

Comment:
why do you need host permissions here?

Reply:
It's a file renamer so that's likely why.

Reply:
so `home`, `/media` and `/run/media`, right?

Reply:
That's also fine, just not sure if it really matters for an offline app. I don't have a strong opinion here.